crates/cairo-profiler/src/trace_reader/function_name.rs

```diff
@@ -43,4 +37,41 @@ impl FunctionName {
 
         Self(function_name.to_string())
     }
+
+    /// `contract_name` and `function_name` are always present (in case they are not in trace we just
+    /// set `<unknown>` string).
+    /// `address` and `selector` are optional and set if `--show-details` flag is enabled
+    /// or names are unknown.
+    pub fn from_entry_point_params(
+        contract_name: Option<String>,
+        function_name: Option<String>,
+        contract_address: ContractAddress,
```

Comment:
Shouldn't we drop contract adresses completely?

Reply:
We will in the future (#82), but it is not related to inlining which this PR is for